src/FLIP.ImagingService.Infrastructure/Services/ImagingService.cs

```diff
@@ -187,6 +192,28 @@ public async Task DeleteProject(string xnatProjectId)
                     "XNAT project deletion was successful for project with ID: {@id}. Response: {@response}",
                     xnatProjectId, response);
             }
+
+            var queuedImports = await this.QueuedPacsRequestRepository.GetByProjectId(xnatProjectId);
+
+            foreach (PacsRequest queuedImport in queuedImports)
+            {
+                var importDeleteResponse = await this.RequestBuilder
+                .SetClient(this.HttpClient)
+                .SetUrl($"xapi/dqr/import/queue/{queuedImport.Id}")
+                .SetMethod(HttpMethod.Delete)
+                .AddXnatAuthCookie(token)
+                .SendAsync();
+
+                if (importDeleteResponse.IsSuccessStatusCode)
+                {
+                    this.Logger.LogDebug(
+                        $"XNAT import request deletion was successful for project with ID: {xnatProjectId}. Request with ID: {queuedImport.Id}");
```

Comment:
```
this.Logger.LogDebug(
                        $"XNAT import request deletion was successful for project with ID: {xnatProjectId}. Request with ID: {queuedImport.Id}"
);
```
Github has decided to format this ^ terribly but with long log lines like this, generally reads better to have the end bracket on a new line

Reply:
Also in our .Net repos, we use a logging library called NLog which has it's own string replacement logic. Preferred method of logging this out would be
```suggestion
                        $"XNAT import request deletion was successful for project with ID: @{XnatProjectId}. Request with ID: @{QueuedImportId}",
   xnatProjectId,
   queuedImport.Id
);
```
The benefit of doing it this way over regular string interpolation is that NLog wont just replace the items in the string but also keep the names and values of those params to be potentially used as filters in the logs

Reply:
Thanks this has been done